openevolve/evaluator.py

```
@@ -18,7 +18,9 @@
 import traceback
 
 from openevolve.config import EvaluatorConfig
+from openevolve.database import ProgramDatabase
 from openevolve.evaluation_result import EvaluationResult
+from openevolve.database import ProgramDatabase
```

Ruff (0.11.9)

Redefinition of unused `ProgramDatabase` from line 21

Remove definition: `ProgramDatabase`

(F811)